app/user/rest.py

```diff
@@ -454,6 +457,26 @@ def send_support_email(user_id):
     return jsonify({"status_code": status_code}), 204
 
 
+@user_blueprint.route('/<uuid:user_id>/contact-request', methods=['POST'])
+def send_contact_request(user_id):
+
+    try:
+        contact = ContactRequest(**request.json)
+        user = get_user_by_email(contact.email_address)
+        if not any([not s.restricted for s in user.services]):
+            contact.tags = ['z_skip_opsgenie', 'z_skip_urgent_escalation']
+
+        status_code = Freshdesk(contact).send_ticket()
```

Comment:
Freshdesk + Zendesk Sell clients will raise exceptions coming from `requests` if the request isn't successful, does this controller will handle those properly?

Reply:
you are right, zendesk is dependent on freshdesk being successful.
Let me rework the code so that they are independent upon each other.
The status_code will report failure if one of them fails....